### PR TITLE
Confidential Ledger CI

### PR DESCRIPTION
**Pull request by christothes**
- CIs

### CODE REVIEW

File: sdk/confidentialledger/ci.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+trigger:
+  branches:
+    include:
+    - master
+    - main
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/confidentialledger/
```
Comment (jsquire) [inline sdk/confidentialledger/ci.yml:11]: Should we include the `ci.yaml` file here as well?

**Review by jsquire — APPROVED** [review]

File: sdk/confidentialledger/ci.yml (hunk 1 above)
Comment (jsquire) [reply sdk/confidentialledger/ci.yml:11]: Disregard;  this is my error in reading comprehension.

Comment (christothes) [reply sdk/confidentialledger/ci.yml:11]: this is the top level path.

**Pull request merged**